srcpkgs/python3-dulwich/template

```diff
@@ -1,26 +1,24 @@
 # Template file for 'python3-dulwich'
 pkgname=python3-dulwich
-version=0.20.44
-revision=2
+version=0.20.50
+revision=1
 build_style=python3-module
+make_check_target="dulwich/tests"
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
-depends="python3-urllib3 python3-certifi"
-checkdepends="${depends} python3-gevent python3-gpg gnupg"
+depends="python3-urllib3"
+checkdepends="${depends} python3-pytest python3-gevent python3-gpg gnupg git"
```

Comment:
could this work with chroot-git?

Reply:
Doesn't look like I can set a custom git command for the tests. If it's removed there aren't any tests errors rather they are skipped.

Reply:
you can set PATH in the template: `PATH="/usr/libexec/chroot-git:$PATH"`

Reply:
~20% of the tests which requires git are failing with chroot-git.